pystachio/choice.py

```diff
@@ -0,0 +1,166 @@
+# Choice types: types that can take one of a group of selected types.
+
+import json
+
+from .base import Object
+from .basic import SimpleObject
+from .compatibility import Compatibility
+from .typing import (
```

Comment:
This will need a `tox -e isort-run` to fixup import sort style for Travis CI (new last week).